modules/couchdb/collect.go

```diff
@@ -89,14 +89,10 @@ func (cdb *CouchDB) collectDBStats(collected map[string]int64, ms *cdbMetrics) {
 
 	for _, dbStats := range ms.DBStats {
 		if dbStats.Error != "" {
-			cdb.Error("database '", dbStats.Key, "' doesn't exist")
+			cdb.Warning("database '", dbStats.Key, "' doesn't exist")
```

Comment:
please use Warningf, why we need to concatenate strings to see the message when it could be `"database '%s' doesn't exist"`?

Reply:
ok, i suggest to prefer format functions in the future